### DISCUSSION TITLE
[Bug] Chiacoins November2022

### DISCUSSION THREAD

**Discussion by shooti22** [Chia-Network/chia-blockchain#13943] [Support]
### What happened?

Hello everyone, the total withdrawals from November 2022 have disappeared in my Chia wallet. Including the coins found. No transactions were carried out by me. My wallet version is 1.6.1. Screenshots from November that really took place via the pool are in my possession. Can someone help me there?

### Version

1.6.1

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by HVNC1** [comment]
try to change the derivative index to the maximum possible number and wait for the wallet to perform all the necessary actions

**Comment by shooti22** [comment]
ok...find it. Let´s start. i will give you a Feedback later :)

**Comment by shooti22** [comment]
nopp..it doesn't work

**Comment by shaneo257** [comment]
have you closed down the client, and deleted the wallet DB?

**Comment by shooti22** [comment]
> have you closed down the client, and deleted the wallet DB?

yes

**Comment by shooti22** [comment]
Hello everyone,

the issue is gone. All Coins are found. 

Thx for Support :)
